src/utils/auth.js

```diff
@@ -0,0 +1,40 @@
+import { SERVER__URL } from "./Api";
+
+// SIGNUP
+
+export const checkResponse = (res) => {
+  if (res.ok) {
+    return res.json();
+  } else {
+    console.error(`Error ${res.status}`);
+    return res.status;
```

Comment:
* You need to invoke an error if the response is not `ok` to be able to `catch` errors in blocks `catch`


It should be like that

```js



export const checkResponse = (res) => {
  if (res.ok) {
    return res.json();
  } else {
    return Promise.reject(`Error ${res.status}`);
  }
};
```